src/main/java/org/jabref/gui/util/Theme.java

```diff
@@ -127,11 +260,35 @@ private void addAndWatchForChanges(Scene scene, URL cssFile, FileUpdateMonitor f
         }
     }
 
+    /**
+     * @return the Theme type
+     */
     public Type getType() {
         return type;
     }
 
-    public Path getPath() {
-        return pathToCss;
+    /**
+     * Provides the raw, configured custom CSS location. This should be a file system path, but the raw string is
+     * returned even if it is not valid in some way. For this reason, the main use case for this getter is to
+     * storing or display the user preference, rather than to read and use the CSS file.
+     *
+     * @return the raw configured CSS location
+     */
+    public String getCssPathString() {
+        return cssPathString;
+    }
+
+    /**
+     * This method allows callers to obtain the theme's additional stylesheet.
+     *
+     * @return called with the stylesheet location if there is an additional stylesheet present and available. The
+     * location will be a local URL. Typically it will be a {@code 'data:'} URL where the CSS is embedded. However for
+     * large themes it can be {@code 'file:'}.
+     */
+    public Optional<String> additionalStylesheet() {
```

Comment:
I would rename it to `getAdditionalStylesheet`

Reply:
Will do!

Reply:
This has now been done.

By the way -- is there a good way for me to include the diff inline here, without just copying the diff? Is that something you would expect?

Reply:
You can link to a diff like this, but that's the best I know.

https://github.com/JabRef/jabref/pull/7292/files#diff-949225e5594af6c25bd85b337a78e60bece8ecfbeba42f6c260ed8b92a6a646bR18-R19

Reply:
Does the job, thanks! 😄